components/TextFields/src/MDCTextField.m

```diff
@@ -226,6 +228,25 @@ - (void)updateBorder {
   self.borderView.borderPath = self.borderPath;
 }
 
+#pragma mark - Input Layout Strut Implementation
+
+- (void)setupInputLayoutStrut {
+  self.inputLayoutStrut = [[UILabel alloc] initWithFrame:CGRectZero];
+  //self.inputLayoutStrut.alpha = 0.f;
```

Comment:
nit: This line is duplicated below.  Remove?

Reply:
Oops. That's from testing.

Reply:
Done.